pkg/util/git.go

```diff
@@ -101,3 +102,18 @@ func LatestTagCommitHash(repo *git.Repository) (*string, *plumbing.Hash, error)
 	return &latestVersionTag, &latestVersionHash, errors.Wrap(err, "error searching git history for latest tag")
 
 }
+
+// GetLatestParentCommit returns the most recent parent commit
+func GetLatestParentCommit(commit *object.Commit) (*object.Commit, error) {
+	var recentParentCommit *object.Commit
```

Comment:
should we print a warning if `NumParents > 1`?

Reply:
sounds good!